tests/generation/Bears.py

```diff
@@ -114,7 +113,7 @@ def setUp(self):
         os.makedirs(self.project_dir, exist_ok=True)
         self.arg_parser = _get_arg_parser()
         self.printer = ConsolePrinter()
-        self.log_printer = LogPrinter(self.printer)
+        self.log_printer = None
```

Comment:
I guess setting `log_printer` to `None` everywhere is leading to failure of CIs. Maybe try replacing them `logging.get_logger` as well?

Reply:
Wrong guess I am afraid. The same problem occurs on master.